### PR TITLE
[ADD] intrastat_base tests

### PR DESCRIPTION
**Pull request by thomaspaulb**
Added tests.

### CODE REVIEW

File: intrastat_base/tests/test_all.py [hunk 1]
```diff
@@ -0,0 +1,27 @@
+# -*- coding: utf-8 -*-
+
+from openerp.tests.common import TransactionCase
+
+
+class TestIntrastatBase(TransactionCase):
+    """Tests for this module"""
+
+    def setUp(self):
+        super(TestIntrastatBase, self).setUp()
```
Comment (hbrunn) [inline intrastat_base/tests/test_all.py]: this is a noop

**Review by hbrunn — APPROVED** [review]
only a small nag

**Comment by pedrobaeza** [comment]
Why tests are not in their respective modules?

**Review by astirpe — APPROVED** [review]
The setUp() method should be removed. Other than that LGTM.

**Comment by thomaspaulb** [comment]
@astirpe @hbrunn Thanks and done.
@pedrobaeza Because the functionality tested is in this module, not in others. It's just to increase coverage.

**Pull request merged**